extensions/empty-screenshots/src/index.ts

```diff
@@ -34,7 +62,11 @@ export default async function main() {
       const stats = await stat(screenshot);
 
       if (stats.birthtimeMs <= priorDate.getTime()) {
-        await rm(screenshot);
+        if (screenshotBehavior === "0") {
+          await rm(screenshot);
+        } else {
+          await moveToTrash(screenshot);
+        }
```

Comment:
🍹 In case we might want to add another behavior in the future, could we augment this a little bit to provide more extensibility?
```suggestion
        switch(screenshotBehavior) {
          case '1':
            return await moveToTrash(screenshot);
          case '0':
          default:
            return await rm(screenshot);          
        }
```